handshake test and integration test fails with go test ./...

When running go test ./... handshake test and integration test fails in Mac OSX and go 1.8.1

```
MacBook-Air:quic-go dineshappavoo$ go test ./...
# github.com/lucas-clemente/quic-go/integrationtests
integrationtests/drop_test.go:10:2: cannot find package "github.com/lucas-clemente/quic-clients" in any of:
	/usr/local/go/src/github.com/lucas-clemente/quic-clients (from $GOROOT)
	/Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-clients (from $GOPATH)
FAIL	github.com/lucas-clemente/quic-go/integrationtests [setup failed]
ok  	github.com/lucas-clemente/quic-go	23.858s
ok  	github.com/lucas-clemente/quic-go/ackhandler	0.064s
ok  	github.com/lucas-clemente/quic-go/congestion	0.049s
ok  	github.com/lucas-clemente/quic-go/crypto	2.896s
?   	github.com/lucas-clemente/quic-go/example	[no test files]
?   	github.com/lucas-clemente/quic-go/example/client	[no test files]
ok  	github.com/lucas-clemente/quic-go/flowcontrol	0.050s
ok  	github.com/lucas-clemente/quic-go/frames	0.096s
ok  	github.com/lucas-clemente/quic-go/h2quic	1.072s
Running Suite: Handshake Suite
==============================
Random Seed: 1492080324
Will run 192 of 192 specs

•••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••
------------------------------
• Failure [0.001 seconds]
Server Config
/Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-go/handshake/server_config_client_test.go:247
  Reading values fromt the TagMap
  /Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-go/handshake/server_config_client_test.go:246
    EXPY
    /Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-go/handshake/server_config_client_test.go:245
      parses the expiry date [It]
      /Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-go/handshake/server_config_client_test.go:225

      Expected
          <int>: 6
      to equal
          <int>: 7

      /Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-go/handshake/server_config_client_test.go:224
------------------------------
••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••

Summarizing 1 Failure:

[Fail] Server Config Reading values fromt the TagMap EXPY [It] parses the expiry date
/Users/dineshappavoo/go/src/github.com/lucas-clemente/quic-go/handshake/server_config_client_test.go:224

Ran 192 of 192 Specs in 0.025 seconds
FAIL! -- 191 Passed | 1 Failed | 0 Pending | 0 Skipped --- FAIL: TestQuicGo (0.03s)
FAIL
FAIL	github.com/lucas-clemente/quic-go/handshake	0.076s
ok  	github.com/lucas-clemente/quic-go/integrationtests/proxy	0.320s
ok  	github.com/lucas-clemente/quic-go/protocol	0.092s
ok  	github.com/lucas-clemente/quic-go/qerr	0.021s
ok  	github.com/lucas-clemente/quic-go/utils	0.025s
```

1. Integration test setup failed
2. handshake test:
 When we update the `Expect(day).To(Equal(6))`  it passes. Byte value could different for the env?

--- a/handshake/server_config_client_test.go
+++ b/handshake/server_config_client_test.go
@@ -85,7 +85,7 @@ var _ = Describe("Server Config", func() {
 		})
 	})
 
-	Context("Reading values fromt the TagMap", func() {
+	Context("Reading values from the TagMap", func() {
 		var scfg *serverConfigClient
 
 		BeforeEach(func() {
@@ -218,7 +218,7 @@ var _ = Describe("Server Config", func() {
 				tagMap[TagEXPY] = []byte{0xdc, 0x89, 0x0e, 0x59, 0, 0, 0, 0} // UNIX Timestamp 0x590e89dc = 1494125020
 				err := scfg.parseValues(tagMap)
 				Expect(err).ToNot(HaveOccurred())
-				year, month, day := scfg.expiry.Date()
+				year, month, day := scfg.expiry.UTC().Date()
 				Expect(year).To(Equal(2017))
 				Expect(month).To(Equal(time.Month(5)))
 				Expect(day).To(Equal(7))

